src/start/syntax.md

```diff
@@ -1 +1,438 @@
+|:--------:|:---------------------------:|:--------:|:------------------:|:------------:|:---------------------------------------:|
+|    `*`   |    numeric multiplication   |     7    | `8 * 3`            | `24`         |                                         |
+|    `/`   |       `real` division       |     7    | `3.0 / 2.0`        | `1.5`        | operands must be `real`                 |
+|   `div`  |       integer divison       |     7    | `3 div 2`          | `1`          | operands must be `int`                  |
+|   `mod`  |          "modulo"           |     7    | `8 mod 3`          | `2`          | operands must be `int`                  |
+|    `+`   |      numeric addition       |     6    | `3 + 4`            | `7`          |                                         |
+|    `-`   |     numeric subtraction     |     6    | `3 - ~2`           | `5`          | `~` denotes negative numbers, e.g. `~5` |
+|    `^`   |     string combination      |     6    | `"foo" ^ "bar"`    | `"foobar"`   |                                         |
+|   `::`   | list construction ("cons")  |     5    | `1 :: [2,3,4]`     | `[1,2,3,4]`  | right-associative                       |
+|    `@`   | list combination ("append") |     5    | `[1,2] @ [3,4]`    | `[1,2,3,4]`  | right-associative                       |
+
+Except for `::` and `@`, the remaining built-in operators above are left-associative.
+Left-associative operations of equal priority implicitly evaluate from left to right.
+Right-associative operations of equal priority implicitly evaluate from right to left.
+
+|           Expression          |        Implicit Interpretation        | Evaluates To |
+|:-----------------------------:|:-------------------------------------:|:------------:|
+| `3 - ~2 + ~5`                 | `((3 - ~2) + ~5)`                     | `0`          |
+| `1 :: 2 :: 3 :: []`           | `1 :: (2 :: (3 :: []))`               | `[1,2,3]`    |
+| `1 :: [] @ 2 :: [] @ 3 :: []` | `1 :: ([] @ (2 :: ([] @ (3 :: []))))` | `[1,2,3]`    |
+
+#### Boolean Operation
+
+There are three main ones: `andalso`, `orelse` and `not`.
+
+|      Expression      | Evaluates To |                             Notes                             |
+|:--------------------:|:------------:|:-------------------------------------------------------------:|
+| `false andalso true` |    `false`   | `andalso` short-circuits if left operand evaluates to `false` |
+| `true orelse false`  |    `true`    | `orelse` short-circuits if left operand evaluates to `true`   |
+|      `not true`      |    `false`   |                                                               |
+|      `not false`     |    `true`    |                                                               |
+
+There are built-in equality operators: `=` and `<>`. These two operate on *equality types*, which include the built-in types mentioned before — and the structured types that can be made from them — __excluding__ `real`.
```

Comment:
is it worth mentioning function types also?

Reply:
Of course — thank you for pointing that out!